src/machi_cr_client.erl

```diff
@@ -434,6 +437,9 @@ do_append_midtail2([FLU|RestFLUs]=FLUs, Prefix, File, Offset, Chunk,
             Resume = {append, Offset, iolist_size(Chunk), File},
             do_repair_chunk(FLUs, Resume, Chunk, [], File, Offset,
                             iolist_size(Chunk), Depth, STime, S);
+        {error, trimmed} = Err ->
+            %% TODO: nothing can be done
+            {reply, Err, S};
```

Comment:
Agree on AP mode, client retry in CP mode.
I think trimmed files should also be repaired, as all files with same name in other flus should be trimmed as well. Maybe in the same way with current repair, or in another file-trim repair.